src/main/c/ssl.c

```diff
@@ -1228,6 +1228,8 @@ static void ssl_info_callback(const SSL *ssl, int where, int ret) {
 TCN_IMPLEMENT_CALL(jlong /* SSL * */, SSL, newSSL)(TCN_STDARGS,
                                                    jlong ctx /* tcn_ssl_ctxt_t * */,
                                                    jboolean server) {
+    SSL *ssl = NULL;
```

Comment:
We may be able to fix this in the windows build file... Anyway let us look into it after this was merged

> Am 14.10.2015 um 17:16 schrieb Scott Mitchell <notifications@github.com>:
> 
> In src/main/c/ssl.c:
> 
> > @@ -1228,6 +1228,8 @@ static void ssl_info_callback(const SSL *ssl, int where, int ret) {
> >  TCN_IMPLEMENT_CALL(jlong /* SSL * */, SSL, newSSL)(TCN_STDARGS,
> >                                                     jlong ctx /* tcn_ssl_ctxt_t * */,
> >                                                     jboolean server) {
> > +    SSL *ssl = NULL;
> C89/ANSI requires declarations to be up front ... didn't' know we were compatible back to c89.
> 
> —
> Reply to this email directly or view it on GitHub.
>